kubernetes/pom.xml

```diff
@@ -0,0 +1,75 @@
+    </parent>
+    <name>Infinispan Simple Tutorials: Kubernetes</name>
+
+    <properties>
+        <!-- Project name used in the OpenShift -->
+        <fabric8.namespace>myproject</fabric8.namespace>
+        <!-- This let's you debug a local container -->
+        <options.debug>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8787</options.debug>
+        <!-- Not sure, what's your configuration, so let's use IPv4 just in case -->
+        <options.useIPv4>-Djava.net.preferIPv4Stack=true</options.useIPv4>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>fabric8-maven-plugin</artifactId>
+                <version>3.1.11</version>
+                <configuration>
+                    <images>
+                        <image>
+                            <name>infinispan/${project.artifactId}:${project.version}</name>
+                            <alias>infinispan-kubernetes</alias>
+                            <build>
+                                <from>java:8-jre-alpine</from>
+                                <assembly>
+                                    <basedir>/deployments</basedir>
+                                    <descriptorRef>artifact-with-dependencies</descriptorRef>
+                                </assembly>
+                                <entryPoint>
+                                    <shell>java -cp "/deployments/*" ${options.debug} ${options.useIPv4} org.infinispan.tutorial.simple.kubernetes.InfinispanKubernetes</shell>
+                                </entryPoint>
+                                <ports>
+                                    <!-- KUBE_PING requires exposing ping port -->
+                                    <port>8888</port>
+                                </ports>
+                                <env>
+                                    <!-- This is how we tell KUBE_PING in which project it should operate -->
+                                    <OPENSHIFT_KUBE_PING_NAMESPACE>${fabric8.namespace}</OPENSHIFT_KUBE_PING_NAMESPACE>
+                                </env>
+                            </build>
+                        </image>
+                    </images>
+                    <!-- We are not interested in creating any services -->
+                    <ignoreServices>true</ignoreServices>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>resource</goal>
+                            <goal>build</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-embedded</artifactId>
```

Comment:
Probably needs to adjust this later since https://github.com/infinispan/infinispan/pull/4492 was pulled in